backend/controllers/followups.js

```diff
@@ -13,7 +15,11 @@ exports.followup = function (req, res, next, id) {
       if (err) {
         return next(err)
       }
-      if (!followup?.answers?._id) {
+      if (
+        !followup?.answers?._id ||
+        (req?.params?.token && !(followup?.accessToken == req.params.token))
```

Comment:
Je ne suis pas sûr de comprendre cette condition.

J'ai l'impression qu'il faudrait ajouter une ligne du genre

> // line to be adjusted once token are used globally

En l'absence de `token` on veut (ou voudra :thinking: ) que ça redirige.

```suggestion
        (req?.params?.token && followup?.accessToken !== req.params.token)
```

Reply:
Effectivement, bien vu